src/lib/cardiac_mechanics/NewtonSolverUtility.hpp

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file NewtonSolverUtility.hpp
+ * @brief Header file defining the NewtonSolverUtility class for configuring
+ * Newton solvers.
+ */
+
+#ifndef NEWTON_SOLVER_UTILITY_HPP
+#define NEWTON_SOLVER_UTILITY_HPP
+
+#include <iostream>
+
+/**
+ * @class NewtonSolverUtility
+ * @brief Utility class for configuring Newton solvers.
+ * @tparam Scalar The scalar type used in the solver configuration.
+ */
+template <typename Scalar> class NewtonSolverUtility {
```

Comment:
Similarly to the linear solver utility, this class is a bit of a missed opportunity to wrap the Newton loop within an external tool, to spare the user from having to reimplement it every time.